finals-2024-01-15/code/5229-ljupco-jakjovski/src/app/models/client.ts

```diff
@@ -0,0 +1,24 @@
+export interface Movie {
```

Comment:
+5 points for the models, the definitions are mostly ok.
Just note that in typescript `[string]` is not identical to `string[]`.